AlfrescoApp/View Controllers/Sync View Controller/RealmSyncViewController.m

```diff
@@ -29,6 +29,7 @@
 #import "FileFolderCollectionViewCell.h"
 #import "ALFSwipeToDeleteGestureRecognizer.h"
 #import "AlfrescoNode+Sync.h"
+#import "RealmSyncManager+Internal.h"
```

Comment:
Please move the needed methods to RealmSyncManager.h file instead of the RealmSyncManager+Internal.h file.